data_science/specs/cohort_spec.rb

```diff
@@ -0,0 +1,100 @@
+require 'rspec'
+require 'json'
+require_relative '../cohort.rb'
+
+RSpec.describe 'Cohort' do
+  let (:test_data) { JSON.parse(File.read('../test_data.json')) }
+  let (:cohort_a) { test_data.select { |v| v['cohort'] == 'A' } }
+
+  before :each do
+    @cohort = Cohort.new(name: 'A', data: cohort_a)
+  end
+
+  describe '#new' do
+    it 'returns a new Cohort object' do
+      expect(@cohort).to be_an_instance_of Cohort
```

Comment:
This test is just silly. Of course `#new` returns an instance of `Cohort`, and if it doesn't, you should use a different language (because Ruby just broke its own semantics) or fire the previous developer (because he just broke the semantics).